### ISSUE TITLE
Extension doesn't install on Windows

### ISSUE THREAD

**Issue by IAmMichellis** [inkstitch/inkstitch#470]
I'm trying to get this running for my mom on her computer: apologies if I'm doing anything silly, this isn't my native environment.

I don't see the extension showing up in the extensions menu in Inkscape.

I followed the instructions [here](https://inkstitch.org/docs/install/#1-download), using the en-US windows tarball, and a brand new install of Inkscape which I grabbed from the Windows store.  

Here's what I did:
- I downloaded and installed Inkscape
- downloaded the latest en-US stitch for Windows tarball
- unzipped it in <userDir>\AppData\Roaming\inkscape\extensions (which I verified was the user extension directory through Inkscape's preferences - note that I had to create the directory because it was a clean install of Inkscape)
- restarted Inkscape
- looked for anything that looked like "stitch" or "ink/stitch" under the extensions menu, but didn't find anything

Other things I tried:
- restarting the computer
- removing the files and installing v1.24 from [here ](https://github.com/inkstitch/inkstitch/releases/tag/v1.24.0)

I'm running Inkscape 0.92.3 and Windows 10 Home.

I went looking for an error log somewhere in AppData but I couldn't find one: please let me know if I can provide any further info :)

**Comment by jameskolme** [comment]
It should work just fine, but so many times not, don´t know why. 
> note that I had to create the directory because it was a clean install of Inkscape

Newer done this, with clean install, directory still exist every time.

What if, extract inkstitch files to actual extensions folder C:\Program Files\Inkscape\share\extensions 
It works from this location too.

**Comment by wwderw** [comment]
I've never tried Inkscape from the store, only used the binary archive (portable) version in my Windows VMs.

Did you run the program first, verified the location and then added the files or did you add the files first and then verified the location?

Main reason that I ask is that if you ran Inkscape for the first time before extracting the extension files, it should have already generated those folders, you shouldn't have had to create them yourself.  I have to wonder if something went wonky there.

I would try the the secondary location that @jameskolme mentioned.  I tend to use that location with the portable version as I'm able to take Inkscape (and the extension) with me on a USB.

**Comment by IAmMichellis** [comment]
Got it!

I messed around with file ownership and corrupted my Inkscape installation, so I don't know if @jameskolme  's suggestion worked because I wasn't able to run Inkscape. So I uninstalled and installed it from the exe download on Inkscape's website (thanks @wwderw ), instead of through the store.  Didn't have to do anything else as I'd already put the stitch files in the user extension directory (but I do see the expected error log and other files there too now, so that's nice :D )

Hopefully it helps someone to know that Inkscape installation through the store doesn't work as expected 🤷‍♀ . Thanks for your help!
